# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../parted/hotfix.patch.musl
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- parted-3.2/libparted/fs/xfs/platform_defs.h.vanilla	2019-02-06 20:10:45.269003633 +0000
+++ parted-3.2/libparted/fs/xfs/platform_defs.h	2019-02-06 20:10:53.713003537 +0000
@@ -44,6 +44,7 @@
 #include <unistd.h>
 #include <sys/param.h>
 #include <sys/types.h>
+#include <fcntl.h>
 
 #if (__GLIBC__ <= 2) && (__GLIBC_MINOR__ <= 1)
 # define constpp	const char * const *
